src/components/Garden/Feed/ProposalSupport.tsx

```diff
@@ -11,16 +11,25 @@ import { ProposalTypes } from '@/types'
 import { safeDiv } from '@utils/math-utils'
 import { getConnectedAccountCast } from '@utils/vote-utils'
 import { VOTE_NAY, VOTE_YEA } from '@/constants'
+import { ProposalType } from '@/hooks/constants'
 
-function ProposalSupport({ proposal }) {
+type ProposalSupportProps = {
+  proposal: ProposalType
+  isAbstainProposal?: boolean
+}
+
+function ProposalSupport({
+  proposal,
+  isAbstainProposal,
+}: ProposalSupportProps) {
   const theme = useTheme()
   const { config } = useGardenState()
   const { requestToken } = config.conviction
 
   return (
     <div
       css={`
-        margin-bottom: ${2 * GU}px;
+        margin-bottom: ${isAbstainProposal ? 0 : 2 * GU}px;
       `}
     >
       <div
```

Comment:
What you think about convert that to styled component? if not its okay for too, just wonder if can bring more component style and get more readability to it

Reply:
You are right 👌 I will change it.

Reply:
I was talking about the div below, but cant tag/mark them here line 32-39. But its awesome too.